bin/application

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+PROCESS=
+CMD_ARGS=
+CMD=
+
+ApplicationUsage () {
+  echo "usage: application [ start | build | server | test | tdd | ci | help | -- COMMAND ]"
+}
+
+ApplicationParseOpts () {
+  PROCESS=$1
+  shift
+  CMD_ARGS=$*
+}
+
+ApplicationRun () {
+  CMD=
+  if [ "$PROCESS" = "help" ] ; then
+    ApplicationUsage
+    exit
+  elif [ "$PROCESS" = "--" ] ; then
```

Comment:
This is the only slightly useful part of this, besides some consistency with our other containers.